<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IQAnalyzePlot</class>
 <widget class="QWidget" name="IQAnalyzePlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IQAnalyzePlot</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="mainVerticalLayout">
     <item>
      <layout class="QHBoxLayout" name="statusHorizontalLayout">
       <item>
        <widget class="QLabel" name="deviceNameLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>AD9361-ZYNQ(192.168.31.18):8088/RX</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="plotStartPushButton">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="plotStopPushButton">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="plotGridLayout">
       <item row="0" column="0">
        <widget class="XxwCustomPlot" name="realTimePlot" native="true"/>
       </item>
       <item row="0" column="1">
        <widget class="XxwCustomPlot" name="spectrumPlot" native="true"/>
       </item>
       <item row="1" column="0">
        <widget class="XxwCustomPlot" name="lofarPlot" native="true"/>
       </item>
       <item row="1" column="1">
        <widget class="XxwCustomPlot" name="constellationPlot" native="true"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XxwCustomPlot</class>
   <extends>QWidget</extends>
   <header>XxwCustomPlot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
